cmd/root.go

```diff
@@ -197,3 +214,32 @@ func fullKeyID(accountID string, keyID string, region string) (string) {
 	baseString := fmt.Sprintf("arn:aws:kms:%s:%s:key/%s", region, accountID, keyID)
 	return baseString
 }
+
+func healthCheck(vaultAddr string) int {
+	checkInterval := os.Getenv("CHECK_INTERVAL")
+	if checkInterval == "" {
+		checkInterval = "10"
+	}
+	checkIntervalNumber, strConfErr := strconv.Atoi(checkInterval)
+	checkError(strConfErr)
+
+	checkWaitTime := time.Duration(checkIntervalNumber) * time.Second
+
+	for {
```

Comment:
It's unclear to me, what's breaks us out of this loop? Is that the `return response.StatusCode`?

Reply:
NVM, I understand it now.